cns/middlewares/swiftV2.go

```diff
@@ -58,19 +120,20 @@ func (m *SWIFTv2Middleware) ValidateIPConfigsRequest(ctx context.Context, req *c
 		mtpnc := v1alpha1.MultitenantPodNetworkConfig{}
```

Comment:
this is valid only for AKS scenario

Reply:
yes, but this function is private only for this swift v2 middleware. If you look at the `IPConfigsHandlerMiddleware` interface, you can see that it is only asking for the `IPConfigsRequestHandlerWrapper` function, in which I have implemented as generic as possible for other possible middleware implementation. Let me know if you have any other concern.